featurebyte/service/feature_store_warehouse.py

```diff
@@ -159,7 +159,7 @@ async def list_columns(
         database_name: str,
         schema_name: str,
         table_name: str,
-    ) -> List[ColumnSpec]:
+    ) -> List[ColumnInfo]:
```

Comment:
Should we use `ColumnSchema` in `featurebyte/session/base.py` as `entity_id`, `critical_data_info` and `semantic_id` will be always empty?

Reply:
We do use ColumnSchema in session/base.py
Do you mean return in here as well?

Reply:
Yes. Sorry for the confusing comment, edited the comment to make it clearer.